.github/workflows/auto-cleanup-bot.yml

```
@@ -47,7 +47,9 @@ jobs:
           author: mdn-bot <108879845+mdn-bot@users.noreply.github.com>
           committer: mdn-bot <108879845+mdn-bot@users.noreply.github.com>
           body: |
-            All issues auto-fixed
+            (This PR was generated by [this run](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}) of the [`auto-cleanup-bot`](https://github.com/${{ github.repository }}/blob/${{ github.ref_name }}/.github/workflows/auto-cleanup-bot.yml) workflow.)
```

What do you reckon? Same comment as below

```suggestion
            This PR was generated in [run ${{ github.run_id }}](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}) of the [`auto-cleanup-bot`](https://github.com/${{ github.repository }}/blob/${{ github.ref_name }}/.github/workflows/auto-cleanup-bot.yml) workflow.
```

---

What's the rationale behind displaying the run id, rather than just linking to it? So that it appears in the edit history of the PR description?

---

I guess it's probably really long, the original is better, but we can drop the parens